### DISCUSSION TITLE
Echelon Sport Connect bike fit

### DISCUSSION THREAD

**Discussion by eMadman** [doudar/SmartSpin2k#313] [Ideas]
Ongoing effort to mod the Schwinn Mod's bike mount into fitting the Echelon Sport connect

I have pretty accurate measurements of the adjustable head tube of the bike from a user and was able to get a good attempt in at reproducing the profile of it.  The user has a 3d printer and has offered to help test it, so getting the fit right shouldn't take too much back and forth effort

From this: 
![272352987_519529289591246_693168597083823718_n](https://user-images.githubusercontent.com/24726844/150838946-61b20ce7-4947-4099-b308-8b23671e69e6.jpg)

To this: 
![image](https://user-images.githubusercontent.com/24726844/150838360-1a0d2d7b-0640-40be-895f-c9c2e4b000fc.png)

I'm also tempted to explore mounting it to the down tube so that users don't have to adjust the mount's placement whenever they raise or lower the handlebar.  This is a bit trickier to measure for and get the right radius.

**Comment by doudar** [comment]
Good work as usual!

**Comment by eMadman** [comment]
This is mostly done and now on the repo in the direct mount mod folder.  The exercise also allowed me to make better CAD that should allow easier adoption to other bikes.  The prong section remains the same for all bikes (where feasible) and what happens behind can be created off new sketches and finally merged back in with the prong section to generate the STL.

![image](https://user-images.githubusercontent.com/24726844/152818141-7ee415d3-c59a-47d1-aa4c-9450345acf82.png)
